fetch.bs

```diff
@@ -4477,19 +4477,27 @@ steps:
 
  <li><p>Let <var>response</var> be null.
 
- <li><p>If <var>request</var>'s <a>local-URLs-only flag</a> is set and <var>request</var>'s
- <a for=request>current URL</a> is not <a lt="is local">local</a>, then set <var>response</var> to a
- <a>network error</a>.
+ <li>
+   <p>If <var>fetchParams</var>'s <a for="fetch params">preloaded response candidate</a> is null:
```

Comment:
Still not using single-space indentation.

Reply:
Ah whoops, should be fixed now.